### ISSUE TITLE
When camCollision is disabled, camInitDir is initially ineffective.

### ISSUE THREAD

**Issue by eugle** [pmndrs/ecctrl#86]
`
<Ecctrl
						...
						camInitDir={{ x: 0.4, y: 0 }}
						
						camCollision={false}>
						<EcctrlAnimation
							characterURL='/models/tribal.gltf'
							animationSet={animationSet}>
							<Character position={[0, -0.9, 0]} />
						</EcctrlAnimation>
					</Ecctrl>
`

When camCollision is disabled, camInitDir is initially ineffective.

Upon the first load, the specified camInitDir angle is not applied, and the default camera view is used instead. When the user tries to adjust the camera using the mouse, it immediately switches to the angle set by camInitDir, which feels awkward for the user.

Thank you for your package, it's excellent.

**Comment by ErdongChen-Andrew** [comment]
https://github.com/user-attachments/assets/bad73f58-c512-40af-bee5-6f927aa7d1af

Works fine on my end 🤔

**Comment by eugle** [comment]
@ErdongChen-Andrew Thank you for your response. Here's the situation: when you use camInitDir={{ x: 0, y: 1 }}, everything works fine, just like in your demo video. However, when you use camInitDir={{ x: 1, y: 0 }}, the issue I mentioned occurs. Additionally, I found that when I set camInitDir={{ x: 3, y: 0 }}, the viewport cannot move up and down. Could you try it and see? It's quite strange.

**Comment by ErdongChen-Andrew** [comment]
Good point, @eugle! This is a really interesting bug though. It should be fixed in the new version 1.0.87. Give it a try.

**Comment by eugle** [comment]
Looking forward to it. Thank you for your patient response.
